### PR TITLE
Update the libraries which are included in the Pi image

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Based on running the following snippet in a [qemu-enabled copy](https://studentrobotics.slack.com/archives/C012NKR0N5C/p1665775486642219?thread_ts=1665774281.869879&cid=C012NKR0N5C) of the built image:
```
python3 -c 'import pkg_resources; print(n.join(str(x) for x in pkg_resources.working_set))'
```

### CODE REVIEW

**Comment by WillB97** [comment]
FYI in [SR2023.0.0rc3](https://github.com/srobo/robot-image/releases/tag/v2023.0.0rc3) the package versions are now:

```
sr.robot3 2023.0.4
pytz 2022.1
setuptools 59.5.0
Pillow 9.0.1
click 8.1.2
PyYAML 6.0
PyGObject 3.42.0
typing-extensions 3.10.0.0
matplotlib 3.5.1
pyserial 3.5
pyparsing 3.0.7
python-dateutil 2.8.2
udiskie 2.3.3
tomli 2.0.1
numpy 1.22.3
six 1.16.0
pydantic 1.9.1
cached-property 1.5.2
pandas 1.4.2
pyusb 1.2.1
j5 1.1.2
tomli-w 1.0.0
pyudev 0.23.2
cycler 0.11.0
astoria 0.10.0
pyquaternion 0.9.9
zoloto 0.8.0
RPi.GPIO 0.7.0
gmqtt 0.6.11
docopt 0.6.2
iniparse 0.5
j5-zoloto 0.4.0
kchd 0.4.0
dbus-next 0.2.3
gpg 1.17.1-unknown
UNKNOWN 0.0.0
```

**Review by trickeydan — APPROVED** [review]

File: programming/python/libraries.md [hunk 1]
```diff
@@ -11,43 +11,41 @@ Robot Kit
...
-* [pyyaml 5.4.1](https://pypi.org/project/PyYAML/)
-* [rpm 4.16.1.3](https://pypi.org/project/rpm)
-* [setuptools 57.4.0](https://pypi.org/project/setuptools)
+* [pyyaml 6.0](https://pypi.org/project/PyYAML)
+* [rpi.gpio 0.7.0](https://pypi.org/project/rpi.gpio)
+* [setuptools 59.5.0](https://pypi.org/project/setuptools)
 * [six 1.16.0](https://pypi.org/project/six)
-* [sr-robot3 2022.0.2](https://pypi.org/project/sr.robot3/)
-* [toml 0.10.2](https://pypi.org/project/toml)
+* [sr.robot3 2023.0.4](https://pypi.org/project/sr.robot3)
```
Comment (RealOrangeOne) [inline programming/python/libraries.md:44]: I believe we bumped this to `.5` to fix the wheel dependency issue

Comment (sedders123) [reply programming/python/libraries.md:44]: Looks like its still `.4` in `robot-image` https://github.com/srobo/robot-image/tree/main/sources/meta-srobo/recipes-robot/python-sr-robot3

Comment (sedders123) [reply programming/python/libraries.md:44]: I thought `.5` only fixed a Windows issue?

Comment (RealOrangeOne) [reply programming/python/libraries.md:44]: If `.4` is in the image, then yeah let's keep it as-is. Wasn't sure if a new image had been baked (yes it only fixes an issue on non-linux).

**Pull request merged**